### DISCUSSION TITLE
如何优雅的在yarn workspace中使用taro

### DISCUSSION THREAD

**Discussion by 757566833** [NervJS/taro#13666] [Q & A]
### 这个特性解决了什么问题？
yarn workspace

### 这个 API 长什么样？
npx @tarojs/cli workspace init packages/myApp 名字期望取myApp

这个自动安装依赖会cd进去，具体的api还是希望大佬们定夺，taro并不清楚我安装依赖的方式

但是创建模板事小，rn事大

今天我在尝鲜rn的时候 yarn workspce xxx dev:rn

直接开始安装react-native 这显然不是我希望的 或者能否提供一个不自动安装依赖的参数

**Comment by shinken008** [comment]
执行了 yarn dev:rn 运行才会去安装吧

**Comment by 757566833** [comment]
> 执行了 yarn dev:rn 运行会去安装吧

是啊，但是我执行的时候 并不需要安装啊

后来放弃了，单独起了一个rn的项目

**Comment by shinken008** [comment]
没懂，yarn dev:rn 表示要跑 RN 端了，不安装依赖跑不起来，不想安装为啥要跑这个命令呢
